aws/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -31,6 +31,7 @@ resource "aws_instance" "controllers" {
     volume_type = "${var.disk_type}"
     volume_size = "${var.disk_size}"
     iops        = "${var.disk_iops}"
+    encrypted   = true
```

Comment:
Is this needed for the kubernetes upgrade? Or why is this added?

Reply:
Ported from:
https://github.com/poseidon/typhoon/commit/6db11d5908ff3e41a291e534598e6216af8eeb49